### ISSUE TITLE
Compilation error on example on Ubuntu

### ISSUE THREAD

**Issue by EmilieSD** [jserot/rfsm-light#4]
Trying to build the chrono or dimp example on Ubuntu with everything but the DOT generation, the interface shows an error window with the following message :
![image](https://user-images.githubusercontent.com/57574048/102088170-d5161c80-3e1a-11eb-864d-6fbfb3d69ab6.png)

--------------
Ubuntu version 18.04.5

- Debug Log with application window open 
```
sudo src/rfsm-light 
Warning: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
Warning: libpng warning: iCCP: extra compressed data
Warning: libpng warning: iCCP: extra compressed data
Warning: libpng warning: iCCP: extra compressed data
Warning: libpng warning: iCCP: extra compressed data
Debug: APPDIR= "/home/.../rfsm-light/src"
Debug: Reading paths from file  "/home/.../rfsm-light/src/rfsm-light.ini"
Debug: "COMPILER" = "rfsmc"
Debug: "DOTPROGRAM" = "dot"
Debug: "DOTVIEWER" = "graphviz"
Debug: "INITDIR" = ""
Debug: "VCDVIEWER" = "gtkwave"
Debug: compile command:  "rfsmc  -ctask -target_dir . /home/.../rfsm-light/examples/dimp/dimp.fsm"
```
- When running the compile command from shell
```
rfsmc  -ctask -target_dir . /home/.../rfsm-light/examples/dimp/dimp.fsm
-------------------------------------------------------------------------------------------------
Reactive Finite State Machine compiler and simulator, version 1.0
-------------------------------------------------------------------------------------------------
File "/home/em/temps_reel/rfsm-light/examples/dimp/dimp.fsm", line 11, characters 2-3:
>  | E2 -> E2 on h when k>0 with k:=k-1  | E1 -> E1 on h when e=1 with k:=k+1  | E2 -> E0 on h when k=0 with s:=0  | E1 -> E2 on h when e=0 with k:=2*k-1,s:=1  | E0 -> E1 on h when e=1 with k:=1;
>  ^
Syntax error
```
... and same type of error for the other example and other build modes.

**Comment by jserot** [comment]
Rfsm-light requires `rfsmc` v >= 1.6 (the 1.0 is obsolete).
Upgrade can be carried out using `opam` :
```
opam update
opam upgrade rfsm
```

**Comment by EmilieSD** [comment]
Can't upgrade to v. 1.6 :

```
$opam update
=-=- Updating package repositories =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[default] synchronized from https://opam.ocaml.org/1.2.2
```

```
$opam upgrade rfsm
Already up-to-date.
```

```
$opam show rfsm
             package: rfsm
             version: 1.0
          repository: default
        upstream-url: http://cloud.ip.univ-bpclermont.fr/~serot/rfsm/dist/rfsm.tar.gz
       upstream-kind: http
   upstream-checksum: e4d7f1086eb346e7a61760036f059bc6
            homepage: http://cloud.ip.uca.fr/~serot/rfsm/
         bug-reports: jocelyn.serot@uca.fr
            dev-repo: git://github.com/jserot/rfsm.git
              author: Jocelyn Sérot <jocelyn.serot@uca.fr>
             license: MIT
             depends: camlp4 & ocamlfind & ocamlbuild & ocamlgraph & menhir & lascar
   installed-version: 1.0 [system]
   available-version: 1.0
         description: A toolset for describing and simulating StateChart-like state diagrams.
```

**Comment by jserot** [comment]
Seems you have opam 1.2.

1.6.0 is an opam 2 package.
You have to upgrade to opam 2.0 :

http://opam.ocaml.org/doc/Install.html
